2018-06-27 00:09:16.809 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:09:16.815 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:10:16.810 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:10:16.816 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:11:16.810 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:11:16.814 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:11:31.354 29909 DEBUG oslo_concurrency.lockutils [req-9c604d4a-1394-48a9-996a-de79aad5c91e 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:11:31.971 29909 DEBUG neutron.agent.dhcp.agent [req-3a9876f3-df93-4f1a-b395-41486dbe7efe - - - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:11:31.974 29909 DEBUG neutron.agent.linux.utils [req-3a9876f3-df93-4f1a-b395-41486dbe7efe - - - - -] Unable to access /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:11:31.975 29909 DEBUG neutron.agent.linux.dhcp [req-3a9876f3-df93-4f1a-b395-41486dbe7efe - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-45850e49-e929-4628-ab2f-85b0830c278b on network 45850e49-e929-4628-ab2f-85b0830c278b does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1267
2018-06-27 00:11:31.976 29909 DEBUG neutron.agent.linux.dhcp [req-3a9876f3-df93-4f1a-b395-41486dbe7efe - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-45850e49-e929-4628-ab2f-85b0830c278b on network 45850e49-e929-4628-ab2f-85b0830c278b does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1288
2018-06-27 00:11:34.853 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:34.878 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:34.898 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:34.923 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:34.941 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:34.960 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', 'link', 'set', 'tapa8a27541-3b', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:34.991 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-o', 'link', 'show', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.016 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-27 00:11:35.413 29909 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tapa8a27541-3b) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 00:11:35.415 29909 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-06-27 00:11:35.416 29909 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tapa8a27541-3b) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 00:11:35.419 29909 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'a8a27541-3b3a-42c6-b0f4-047e0aa41434', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:26:0e:c5'})), record=tapa8a27541-3b) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 00:11:35.465 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapa8a27541-3b', 'address', 'fa:16:3e:26:0e:c5'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.485 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.502 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tapa8a27541-3b', 'netns', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.538 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', 'link', 'set', 'tapa8a27541-3b', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.557 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', 'link', 'set', 'tapa8a27541-3b', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.576 29909 DEBUG oslo_concurrency.lockutils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Acquired semaphore "iptables-qdhcp-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:11:35.578 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.607 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.634 29909 DEBUG neutron.agent.linux.iptables_manager [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-06-27 00:11:35.636 29909 DEBUG oslo_concurrency.lockutils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Releasing semaphore "iptables-qdhcp-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:11:35.638 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-o', 'link', 'show', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.657 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', 'addr', 'show', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.675 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'addr', 'add', '11.105.1.2/24', 'scope', 'global', 'dev', 'tapa8a27541-3b', 'brd', '11.105.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.696 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.716 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Setting IPv4 gateway for dhcp netns on net 45850e49-e929-4628-ab2f-85b0830c278b to 11.105.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1164
2018-06-27 00:11:35.718 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.105.1.1', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.736 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.753 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Cleaning stale devices for network 45850e49-e929-4628-ab2f-85b0830c278b _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1369
2018-06-27 00:11:35.754 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.768 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.787 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:623
2018-06-27 00:11:35.789 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/leases with contents:
1530137495 fa:16:3e:26:0e:c5 11.105.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:651
2018-06-27 00:11:35.791 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:11:35.793 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:11:35.798 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Unable to access /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:11:35.801 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host', '--addn-hosts=/var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tapa8a27541-3b', '--dhcp-range=set:tag0,11.105.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:35.843 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:11:35.846 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:11:35.847 29909 DEBUG neutron.agent.linux.external_process [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:11:35.848 29909 DEBUG oslo_concurrency.lockutils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:11:35.851 29909 DEBUG oslo_concurrency.lockutils [req-9c604d4a-1394-48a9-996a-de79aad5c91e 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:11:37.021 29909 DEBUG neutron.agent.dhcp.agent [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:11:37.024 29909 DEBUG neutron.agent.linux.dhcp [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:11:37.029 29909 DEBUG neutron.agent.linux.dhcp [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:11:37.035 29909 DEBUG neutron.agent.linux.utils [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:37.059 29909 DEBUG neutron.agent.linux.dhcp [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:11:37.061 29909 DEBUG neutron.agent.linux.utils [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:37.083 29909 DEBUG neutron.agent.linux.utils [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:37.102 29909 DEBUG neutron.agent.linux.utils [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:11:37.104 29909 DEBUG neutron.agent.linux.utils [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:11:37.104 29909 DEBUG neutron.agent.linux.external_process [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:11:37.105 29909 DEBUG oslo_concurrency.lockutils [req-f09359a8-e921-47ea-9327-43cb65dffd65 - - - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:11:37.111 29909 DEBUG oslo_concurrency.lockutils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:11:37.112 29909 DEBUG neutron.agent.dhcp.agent [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T22:11:32Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-45850e49-e929-4628-ab2f-85b0830c278b, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.2'}], id=a8a27541-3b3a-42c6-b0f4-047e0aa41434, mac_address=fa:16:3e:26:0e:c5, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=False, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:11:33Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:409
2018-06-27 00:11:37.113 29909 DEBUG oslo_concurrency.lockutils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:11:37.117 29909 DEBUG oslo_concurrency.lockutils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:11:37.118 29909 INFO neutron.agent.dhcp.agent [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-26T22:11:32Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-45850e49-e929-4628-ab2f-85b0830c278b, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.2'}], id=a8a27541-3b3a-42c6-b0f4-047e0aa41434, mac_address=fa:16:3e:26:0e:c5, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=False, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:11:34Z
2018-06-27 00:11:37.120 29909 DEBUG neutron.agent.dhcp.agent [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:11:37.122 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:11:37.125 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:11:37.130 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:37.165 29909 DEBUG neutron.agent.linux.dhcp [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:11:37.169 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:37.191 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:11:37.209 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:11:37.211 29909 DEBUG neutron.agent.linux.utils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:11:37.212 29909 DEBUG neutron.agent.linux.external_process [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:11:37.213 29909 DEBUG oslo_concurrency.lockutils [req-ad757240-f8b6-4e16-a7b4-40fdbca0cce5 - - - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:12:16.810 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:12:16.815 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:12:17.531 29909 DEBUG oslo_concurrency.lockutils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:12:17.532 29909 INFO neutron.agent.dhcp.agent [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T22:12:15Z, description=, device_id=d58b2d94-4d94-4f48-a554-47fa95aa4ec1, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.1'}], id=6a7c68ec-bb4c-4532-a162-0a3dd7784b31, mac_address=fa:16:3e:f2:0a:ca, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=False, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:12:16Z
2018-06-27 00:12:17.534 29909 DEBUG neutron.agent.dhcp.agent [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:12:17.537 29909 DEBUG neutron.agent.linux.dhcp [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:12:17.540 29909 DEBUG neutron.agent.linux.dhcp [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:12:17.546 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:17.567 29909 DEBUG neutron.agent.linux.dhcp [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:12:17.569 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:17.588 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:17.607 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:12:17.608 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:12:17.609 29909 DEBUG neutron.agent.linux.external_process [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:12:17.610 29909 DEBUG oslo_concurrency.lockutils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:12:18.941 29909 DEBUG oslo_concurrency.lockutils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:12:18.943 29909 INFO neutron.agent.dhcp.agent [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T22:12:15Z, description=, device_id=d58b2d94-4d94-4f48-a554-47fa95aa4ec1, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.1'}], id=6a7c68ec-bb4c-4532-a162-0a3dd7784b31, mac_address=fa:16:3e:f2:0a:ca, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=False, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:12:16Z
2018-06-27 00:12:18.945 29909 DEBUG neutron.agent.dhcp.agent [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:12:18.947 29909 DEBUG neutron.agent.linux.dhcp [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:12:18.949 29909 DEBUG neutron.agent.linux.dhcp [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:12:18.956 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:18.981 29909 DEBUG neutron.agent.linux.dhcp [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:12:18.983 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:18.998 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:19.016 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:12:19.018 29909 DEBUG neutron.agent.linux.utils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:12:19.020 29909 DEBUG neutron.agent.linux.external_process [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:12:19.022 29909 DEBUG oslo_concurrency.lockutils [req-b5815808-f0be-400e-9d42-fdcf87634322 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:12:22.232 29909 DEBUG oslo_concurrency.lockutils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:12:22.233 29909 INFO neutron.agent.dhcp.agent [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T22:12:15Z, description=, device_id=d58b2d94-4d94-4f48-a554-47fa95aa4ec1, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.1'}], id=6a7c68ec-bb4c-4532-a162-0a3dd7784b31, mac_address=fa:16:3e:f2:0a:ca, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=False, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:12:18Z
2018-06-27 00:12:22.236 29909 DEBUG neutron.agent.dhcp.agent [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:12:22.238 29909 DEBUG neutron.agent.linux.dhcp [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:12:22.240 29909 DEBUG neutron.agent.linux.dhcp [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:12:22.245 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:22.263 29909 DEBUG neutron.agent.linux.dhcp [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:12:22.266 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:22.283 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:22.300 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:12:22.301 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:12:22.303 29909 DEBUG neutron.agent.linux.external_process [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:12:22.304 29909 DEBUG oslo_concurrency.lockutils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:12:23.106 29909 DEBUG oslo_concurrency.lockutils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:12:23.107 29909 INFO neutron.agent.dhcp.agent [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-26T22:12:15Z, description=, device_id=d58b2d94-4d94-4f48-a554-47fa95aa4ec1, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.1'}], id=6a7c68ec-bb4c-4532-a162-0a3dd7784b31, mac_address=fa:16:3e:f2:0a:ca, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=False, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:12:22Z
2018-06-27 00:12:23.108 29909 DEBUG neutron.agent.dhcp.agent [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:12:23.110 29909 DEBUG neutron.agent.linux.dhcp [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:12:23.112 29909 DEBUG neutron.agent.linux.dhcp [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:12:23.116 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:23.133 29909 DEBUG neutron.agent.linux.dhcp [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:12:23.135 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:23.152 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:12:23.167 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:12:23.169 29909 DEBUG neutron.agent.linux.utils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:12:23.170 29909 DEBUG neutron.agent.linux.external_process [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:12:23.171 29909 DEBUG oslo_concurrency.lockutils [req-b02f22c5-14cf-43e0-a35d-7468f9054713 - - - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:12:32.152 29909 DEBUG oslo_concurrency.lockutils [req-684ebde5-293e-4e67-b6ed-cc15ae60101c 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:12:32.153 29909 DEBUG oslo_concurrency.lockutils [req-684ebde5-293e-4e67-b6ed-cc15ae60101c 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:12:36.598 29909 DEBUG oslo_concurrency.lockutils [req-552e0605-8109-48d7-a6a4-2001de20de64 - - - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:12:36.600 29909 DEBUG oslo_concurrency.lockutils [req-552e0605-8109-48d7-a6a4-2001de20de64 - - - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:12:37.416 29909 DEBUG oslo_concurrency.lockutils [req-552e0605-8109-48d7-a6a4-2001de20de64 - - - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:12:37.418 29909 DEBUG oslo_concurrency.lockutils [req-552e0605-8109-48d7-a6a4-2001de20de64 - - - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:13:15.850 29909 DEBUG oslo_concurrency.lockutils [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:13:15.852 29909 INFO neutron.agent.dhcp.agent [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T22:13:13Z, description=, device_id=9be3661c-5ec6-4507-b455-7b6b01c69c75, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.5'}], id=045f2bc6-77cb-4eae-8aaf-9544496fa91f, mac_address=fa:16:3e:d8:4f:c6, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=True, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=3, security_groups=[u'5f94c186-252f-4531-8b91-ca1e6d0fae27'], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:13:14Z
2018-06-27 00:13:15.854 29909 DEBUG neutron.agent.dhcp.agent [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:13:15.856 29909 DEBUG neutron.agent.linux.dhcp [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:13:15.860 29909 DEBUG neutron.agent.linux.dhcp [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:13:15.867 29909 DEBUG neutron.agent.linux.utils [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:15.888 29909 DEBUG neutron.agent.linux.dhcp [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:13:15.891 29909 DEBUG neutron.agent.linux.utils [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:15.913 29909 DEBUG neutron.agent.linux.utils [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:15.931 29909 DEBUG neutron.agent.linux.utils [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:13:15.933 29909 DEBUG neutron.agent.linux.utils [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:13:15.935 29909 DEBUG neutron.agent.linux.external_process [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:13:15.936 29909 DEBUG oslo_concurrency.lockutils [req-26d33347-5ffe-489f-89a1-3943660999b1 769baeeb603c418dbe470dc33d9425e8 801910ae7da24a5cbe949ae048a826f2 - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:13:16.811 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:13:16.814 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:13:19.717 29909 DEBUG oslo_concurrency.lockutils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:13:19.719 29909 INFO neutron.agent.dhcp.agent [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-06-26T22:13:13Z, description=, device_id=9be3661c-5ec6-4507-b455-7b6b01c69c75, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.5'}], id=045f2bc6-77cb-4eae-8aaf-9544496fa91f, mac_address=fa:16:3e:d8:4f:c6, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=True, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=4, security_groups=[u'5f94c186-252f-4531-8b91-ca1e6d0fae27'], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:13:18Z
2018-06-27 00:13:19.721 29909 DEBUG neutron.agent.dhcp.agent [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:13:19.724 29909 DEBUG neutron.agent.linux.dhcp [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:13:19.728 29909 DEBUG neutron.agent.linux.dhcp [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:13:19.734 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:19.754 29909 DEBUG neutron.agent.linux.dhcp [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:13:19.756 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:19.773 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:19.794 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:13:19.796 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:13:19.799 29909 DEBUG neutron.agent.linux.external_process [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:13:19.800 29909 DEBUG oslo_concurrency.lockutils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:13:20.590 29909 DEBUG oslo_concurrency.lockutils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:13:20.591 29909 INFO neutron.agent.dhcp.agent [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-06-26T22:13:13Z, description=, device_id=9be3661c-5ec6-4507-b455-7b6b01c69c75, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'bbea6c8b-3dde-49b2-85a9-4052574b80a7', u'ip_address': u'11.105.1.5'}], id=045f2bc6-77cb-4eae-8aaf-9544496fa91f, mac_address=fa:16:3e:d8:4f:c6, name=, network_id=45850e49-e929-4628-ab2f-85b0830c278b, port_security_enabled=True, project_id=801910ae7da24a5cbe949ae048a826f2, revision_number=5, security_groups=[u'5f94c186-252f-4531-8b91-ca1e6d0fae27'], status=DOWN, tags=[], tenant_id=801910ae7da24a5cbe949ae048a826f2, updated_at=2018-06-26T22:13:19Z
2018-06-27 00:13:20.592 29909 DEBUG neutron.agent.dhcp.agent [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:13:20.595 29909 DEBUG neutron.agent.linux.dhcp [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:13:20.597 29909 DEBUG neutron.agent.linux.dhcp [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:13:20.603 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:20.622 29909 DEBUG neutron.agent.linux.dhcp [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:13:20.624 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:20.645 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:13:20.666 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:13:20.667 29909 DEBUG neutron.agent.linux.utils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:13:20.668 29909 DEBUG neutron.agent.linux.external_process [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:13:20.670 29909 DEBUG oslo_concurrency.lockutils [req-42bb45b9-e1e0-4e8c-a0ca-c6e57579fe8d 479e046780fe410b9faee29c0cb53b35 3218dc7862004758b6a8f5b0cc5d6ad5 - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:14:11.820 29909 DEBUG oslo_concurrency.lockutils [req-c6e35ab6-1480-4a4c-8cf8-d4ae91e1fa0d 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:14:11.821 29909 DEBUG oslo_concurrency.lockutils [req-c6e35ab6-1480-4a4c-8cf8-d4ae91e1fa0d 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:14:13.290 29909 DEBUG oslo_concurrency.lockutils [req-c6e35ab6-1480-4a4c-8cf8-d4ae91e1fa0d 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:14:13.291 29909 DEBUG oslo_concurrency.lockutils [req-c6e35ab6-1480-4a4c-8cf8-d4ae91e1fa0d 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-6ca5b7c5-23c0-4771-ad3c-ae15aacd634c" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:14:16.811 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:14:16.818 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:15:16.816 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:15:16.826 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.010s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:16:16.812 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:16:16.816 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:17:16.812 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:17:16.817 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:18:16.812 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:18:16.818 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.006s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:18:51.790 29909 DEBUG oslo_concurrency.lockutils [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:18:51.791 29909 DEBUG neutron.agent.dhcp.agent [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:18:51.794 29909 DEBUG neutron.agent.linux.utils [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'dhcp_release', 'tapa8a27541-3b', '11.105.1.5', 'fa:16:3e:d8:4f:c6'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:18:51.820 29909 DEBUG neutron.agent.linux.dhcp [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:18:51.823 29909 DEBUG neutron.agent.linux.dhcp [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:18:51.830 29909 DEBUG neutron.agent.linux.utils [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:18:51.858 29909 DEBUG neutron.agent.linux.dhcp [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:18:51.865 29909 DEBUG neutron.agent.linux.utils [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:18:51.886 29909 DEBUG neutron.agent.linux.utils [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:18:51.908 29909 DEBUG neutron.agent.linux.utils [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:18:51.909 29909 DEBUG neutron.agent.linux.utils [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:18:51.916 29909 DEBUG neutron.agent.linux.external_process [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:18:51.918 29909 DEBUG oslo_concurrency.lockutils [req-3b7f3830-08cd-43fd-9ed8-2a754dfb45ee 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:19:16.812 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:19:16.817 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:19:45.666 29909 DEBUG oslo_concurrency.lockutils [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:19:45.668 29909 DEBUG neutron.agent.dhcp.agent [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:19:45.671 29909 DEBUG neutron.agent.linux.utils [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'dhcp_release', 'tapa8a27541-3b', '11.105.1.1', 'fa:16:3e:f2:0a:ca'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:19:45.696 29909 DEBUG neutron.agent.linux.dhcp [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Building host file: /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:682
2018-06-27 00:19:45.708 29909 DEBUG neutron.agent.linux.dhcp [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Done building host file /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:721
2018-06-27 00:19:45.717 29909 DEBUG neutron.agent.linux.utils [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-HUP', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:19:45.734 29909 DEBUG neutron.agent.linux.dhcp [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Reloading allocations for network: 45850e49-e929-4628-ab2f-85b0830c278b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:511
2018-06-27 00:19:45.736 29909 DEBUG neutron.agent.linux.utils [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-4', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:19:45.758 29909 DEBUG neutron.agent.linux.utils [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b', 'ip', '-6', 'route', 'list', 'dev', 'tapa8a27541-3b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:19:45.780 29909 DEBUG neutron.agent.linux.utils [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:19:45.782 29909 DEBUG neutron.agent.linux.utils [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:19:45.783 29909 DEBUG neutron.agent.linux.external_process [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:19:45.784 29909 DEBUG oslo_concurrency.lockutils [req-3d587d93-54de-4659-ae9f-0e9c8ef2d161 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:20:15.510 29909 DEBUG oslo_concurrency.lockutils [req-7d04fbc8-af81-44a1-a6c7-0c79d60967ec 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:20:15.512 29909 DEBUG neutron.agent.dhcp.agent [req-7d04fbc8-af81-44a1-a6c7-0c79d60967ec 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:20:15.516 29909 DEBUG neutron.agent.linux.utils [req-7d04fbc8-af81-44a1-a6c7-0c79d60967ec 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command (rootwrap daemon): ['kill', '-9', '1468'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:20:15.537 29909 DEBUG neutron.agent.linux.utils [req-7d04fbc8-af81-44a1-a6c7-0c79d60967ec 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-06-27 00:20:15.586 29909 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tapa8a27541-3b) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-06-27 00:20:15.685 29909 DEBUG neutron.agent.linux.utils [req-56b221f3-598f-4966-be11-1ba751101756 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:20:15.699 29909 DEBUG neutron.agent.linux.utils [req-56b221f3-598f-4966-be11-1ba751101756 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-45850e49-e929-4628-ab2f-85b0830c278b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:20:15.718 29909 DEBUG oslo_concurrency.lockutils [req-56b221f3-598f-4966-be11-1ba751101756 - - - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:20:16.813 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:20:16.817 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-27 00:20:25.345 29909 DEBUG oslo_concurrency.lockutils [req-906fa328-fc3c-4268-813e-1cfdc356f4c4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:20:25.936 29909 DEBUG neutron.agent.linux.utils [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:20:25.938 29909 DEBUG neutron.agent.linux.utils [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] Unable to access /var/lib/neutron/external/pids/45850e49-e929-4628-ab2f-85b0830c278b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:20:25.939 29909 DEBUG neutron.agent.linux.external_process [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:20:25.941 29909 DEBUG neutron.agent.dhcp.agent [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] Calling driver for network: 45850e49-e929-4628-ab2f-85b0830c278b action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-06-27 00:20:25.943 29909 DEBUG neutron.agent.linux.utils [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] Unable to access /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:20:25.944 29909 DEBUG neutron.agent.linux.utils [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] Unable to access /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-06-27 00:20:25.945 29909 DEBUG neutron.agent.linux.external_process [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] No process started for 45850e49-e929-4628-ab2f-85b0830c278b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-06-27 00:20:25.946 29909 DEBUG neutron.agent.linux.dhcp [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] Impossibile accedere a /var/lib/neutron/dhcp/45850e49-e929-4628-ab2f-85b0830c278b/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:268
2018-06-27 00:20:25.947 29909 DEBUG neutron.agent.linux.dhcp [req-134f3221-7603-49e4-9c59-37d6c80818c6 - - - - -] No interface exists for network 45850e49-e929-4628-ab2f-85b0830c278b destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1482
2018-06-27 00:20:25.991 29909 DEBUG neutron.agent.linux.utils [req-de5746ca-a718-4655-a6d3-1bd658423cbf - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-06-27 00:20:26.008 29909 DEBUG neutron.agent.linux.dhcp [req-de5746ca-a718-4655-a6d3-1bd658423cbf - - - - -] Namespace already deleted: qdhcp-45850e49-e929-4628-ab2f-85b0830c278b _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:246
2018-06-27 00:20:26.009 29909 DEBUG oslo_concurrency.lockutils [req-de5746ca-a718-4655-a6d3-1bd658423cbf - - - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:20:27.533 29909 DEBUG oslo_concurrency.lockutils [req-906fa328-fc3c-4268-813e-1cfdc356f4c4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Acquired semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-06-27 00:20:27.534 29909 DEBUG oslo_concurrency.lockutils [req-906fa328-fc3c-4268-813e-1cfdc356f4c4 48ab82a30a8f41a1a784b8bc09bc487a 5528f1fe5f894ddba5fb2776fb82674e - - -] Releasing semaphore "dhcp-agent-network-lock-45850e49-e929-4628-ab2f-85b0830c278b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-06-27 00:20:29.425 29909 DEBUG neutron.agent.dhcp.agent [-] resync (45850e49-e929-4628-ab2f-85b0830c278b): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:255
2018-06-27 00:20:29.427 29909 INFO neutron.agent.dhcp.agent [-] Synchronizing state
2018-06-27 00:20:30.419 29909 INFO neutron.agent.dhcp.agent [req-452c543e-cda5-4a18-9bd4-ff74a26b0e82 - - - - -] All active networks have been fetched through RPC.
2018-06-27 00:20:30.421 29909 INFO neutron.agent.dhcp.agent [-] Starting network 6ca5b7c5-23c0-4771-ad3c-ae15aacd634c dhcp configuration
2018-06-27 00:20:30.422 29909 INFO neutron.agent.dhcp.agent [-] Finished network 6ca5b7c5-23c0-4771-ad3c-ae15aacd634c dhcp configuration
2018-06-27 00:20:30.422 29909 INFO neutron.agent.dhcp.agent [req-452c543e-cda5-4a18-9bd4-ff74a26b0e82 - - - - -] Synchronizing state complete
2018-06-27 00:21:16.813 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-27 00:21:16.816 29909 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
